cogs/clan.py

```diff
@@ -0,0 +1,130 @@
+from discord.ext import commands
+import discord
+
+import pydest
+import json
+
+from cogs.utils.message_manager import MessageManager
+from cogs.utils import constants, helpers
+
+class Clan:
+    class RewardEntry:
+        name = ""
+        earned = False
+        reward_hash = 0
+        
+        def __init__(self, rhash, earned):
+            self.reward_hash = rhash
+            self.earned = earned
+
+
+        def __str__(self):
+            return "reward hash={} | name={} | earned={}".format(self.reward_hash,self.name, self.earned)
+
+
+        def set_name(self, name):
+            self.name = name
+
+        
+        def __len__(self):
+            return len(self.name)
+
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.group()
+    @commands.cooldown(rate=2, per=5, type=commands.BucketType.user)
+    async def clan(self, ctx, username=None, platform=None):
+        """Display a Destiny 2 clan
+        
+        In order to use this command for your own clan, you must first register your Destiny 2
+        account with the bot via the register command.
+
+        'clan' - Display your clan
+        """
+
+        manager = MessageManager(ctx)
+        await ctx.channel.trigger_typing()
+
+        membership_details = await helpers.get_membership_details(self.bot, ctx, username, platform)
+
+        if isinstance(membership_details, str):
+            await manager.send_message(membership_details)
+            return await manager.clean_messages()
+
+        platform_id, membership_id, display_name = membership_details
+
+        try:
+            res = await self.bot.destiny.api.get_groups_for_member(platform_id, membership_id)
+        except:
+            await manager.send_message("Sorry, I can't seem to retrieve that clan right now")
+            return await manager.clean_messages()
+
+        if res['ErrorCode'] != 1:
+            await manager.send_message("Sorry I can't seem to retrieve that clan right now")
+            return await manager.clean_messages()
+
+        group_id = res['Response']['results'][0]['member']['groupId']
+
+        try:
+            ms = await self.bot.destiny.api.get_weekly_milestones(group_id)
+            ms_def = await self.bot.destiny.api.get_weekly_milestone_definitions(ms['Response']['milestoneHash'])
```

Comment:
`get_weekly_milestone_definitions` is not a part of Pydest, so this line fails. Perhaps you meant to use `get_milestone_definitions` instead?